storefront/features/ProductGrid/ProductGrid.tsx

```diff
@@ -23,6 +22,14 @@ export default ProductGrid;
 const Grid = styled.div`
   width: 100%;
   display: grid;
-  grid-template-columns: 2fr;
-  gap: 16px;
+  grid-template-columns: repeat(3, 1fr);
```

Comment:
:+1: you can use it, do you know caniuse?
https://caniuse.com/?search=grid-template-columns